java/vector/src/main/codegen/templates/UnionListWriter.java

```diff
@@ -178,6 +178,10 @@ public void write(DecimalHolder holder) {
     writer.setPosition(writer.idx()+1);
   }
 
+  public void writeNull() {
```

Comment:
I think the same change is required in UnionFixedSizeListWriter as well.

Reply:
done